src/lib.rs

```diff
@@ -490,24 +507,24 @@ impl Instance {
 
     /// Retrieves an [`Adapter`] which matches the given descriptor.
     ///
-    /// If there are no available adapters matching `desc`, this function will return another
+    /// If there are no available adapters matching `options`, this function will return another
     /// adapter.
     ///
     /// # Panics
     ///
     /// Panics if there are no available adapters. This will occur if none of the graphics backends
     /// are enabled.
-    pub fn get_adapter(&self, options: Option<&RequestAdapterOptions>) -> Adapter {
+    pub fn request_adapter(&self, options: &RequestAdapterOptions) -> Adapter {
```

Comment:
How are you supposed to call request_adapter(None) now?

Reply:
it's equivalent to `request_adapter(&wgpu::RequestAdapterOptions::default())`

Reply:
Oh yeah fair enough.

I guess my only issue is one day when we are using wgpu-rs on the web, we wont be able to access that part of the API, if only for testing different browser implementations, but that seems pretty obscure.

Reply:
The `Option` thing in WebIDL specification itself is defined to be equivalent of supplying a dictionary with the fields full of default values, so it's not like a browser has freedom to do anything other than this (it would be a bug!).